src/tests/ftest/util/mdtest_test_base.py

```diff
@@ -162,27 +160,32 @@ def get_job_manager_command(self, manager):
             JobManager: the object for the mpi job manager command
 
         """
-        # Initialize MpioUtils if mdtest needs to be run using mpich
         if manager == "MPICH":
+            # Initialize MpioUtils if mdtest needs to be run using mpich
             mpio_util = MpioUtils()
             if mpio_util.mpich_installed(self.hostlist_clients) is False:
                 self.fail("Exiting Test: Mpich not installed")
-            path = os.path.join(mpio_util.mpichinstall, "bin")
-            return Mpirun(self.mdtest_cmd, path, mpitype="mpich")
+            manager_class = Mpich(self.mdtest_cmd)
+        else:
+            manager_class = OpenMPI(self.mdtest_cmd)
```

Comment:
(pylint-redefined-variable-type) Redefinition of manager_class type from job_manager_utils.Mpich to job_manager_utils.OpenMPI